docs/contributing/frontend.md

```diff
@@ -221,13 +229,31 @@ import { visuallyHidden } from "@mui/utils";
 </Button>;
 ```
 
-### Should I create a new component?
+### Should I create a new component or module?
+
+Components could technically be used in any codebase and still feel at home. A
+module would only make sense in the Coder codebase.
+
+- Component
+  - Simple
+  - Atomic, used in multiple places
+  - Good Examples: `Badge`, `Form`, `Timeline`
```

Comment:
can we add something like...

- Generic–would be useful as a component outside of the Coder product